timing_model/flow.tcl

```diff
@@ -0,0 +1,120 @@
+# Current script location
```

Comment:
This feels like can be a rewite as a python file, which then can be tested using `pytest`